python/pyspark/tests/test_readwrite.py

```diff
@@ -231,8 +231,9 @@ def test_newhadoop(self):
         )
         self.assertEqual(result, data)
 
+        fmt = "org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat"
         conf = {
-            "mapreduce.job.outputformat.class": "org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat",
+            "mapreduce.job.outputformat.class": fmt,
```

Comment:
I don't feel strongly about it, but brackets should work in such cases, without introducing new variables

```python
    "mapreduce.job.outputformat.class": (
        "org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat"
    ),
```

Reply:
im working on another PR to enable some rules - I will create the Pr today. Let me fix it in the PR together because that Pr touches here and there too..